app-server/app/src/main/scala/thumbnail/Thumbnail.scala

```diff
@@ -0,0 +1,98 @@
+package com.azavea.rf.thumbnail
+
+import java.sql.Timestamp
+
+import scala.concurrent.{Future, ExecutionContext}
+import scala.util.{Success, Failure, Try}
+
+import com.azavea.rf.AkkaSystem
+import com.azavea.rf.datamodel.latest.schema.tables._
+import com.azavea.rf.utils.Database
+
+trait Thumbnail extends AkkaSystem.LoggerExecutor {
+
+  implicit val database:Database
+
+  import database.driver.api._
+
+  /** Insert a thumbnail into the database
```

Comment:
Changing to match surrounding code

Do we have style somewhere? It looks like the two that didn't match _do_ match [Scaladoc's format](http://docs.scala-lang.org/style/scaladoc.html), but we went with the other format everywhere